marklogic-data-hub/src/main/java/com/marklogic/hub/HubConfig.java

```diff
@@ -541,4 +539,12 @@
      * @return
      */
     HubConfig withPropertiesFromEnvironment(String environment);
+
+    /**
+     * Loads HubConfig object with values from gradle.properties (optionally overriden with
+     * gradle-(environment).properties). This method must be called every time after Spring creates
+     * HubConfig object and the project is initialized with createProject(String projectDirString)
+     * method
+     */
```

Comment:
@akshaysonvane , should the doc include something to the effect of "make all changes wanted  to "hubConfig" using set methods and then call refreshProject" ?

Reply:
Updated description 9d653dc